src/deepsparse/benchmark/benchmark_model.py

```diff
@@ -153,25 +167,21 @@ def parse_args():
         default=1,
         help="The batch size to run the analysis for. Must be greater than 0",
     )
-
     parser.add_argument(
         "-seq_len",
         "--sequence_length",
         type=int,
-        default=2048,
-        help="The sequence length to run the "
-        "KV cache supported model benchmarks for. "
-        "Must be greater than 0, default is 2048",
+        default=512,
+        help="The sequence length to run the KV cache supported model "
+        "benchmarks for. Must be 1 <= seq_len, default is 512",
```

Comment:
>Must be 1 <= seq_len,

do normal people write it this way?

Reply:
lol i was making it the same as input_ids_length